themes/bootstrap3/templates/RecordDriver/DefaultRecord/list-entry.phtml

```diff
@@ -33,36 +33,37 @@
     <?php endif; ?>
     <div class="media-body">
       <div class="result-body">
-        <div class="resultItemLine1">
-          <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
-          <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
-            <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
-          <?php if (!$missing): ?></a><?php endif; ?>
-          <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
-            <div class="title-alt">
-              <?=$this->escapeHtml($altTitle)?>
-            </div>
-          <?php endforeach; ?>
-        </div>
+        <header>
+          <h2>
+            <?php $missing = $this->driver instanceof \VuFind\RecordDriver\Missing; ?>
+            <?php if (!$missing): ?><a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>" class="getFull" data-view="<?=$this->params->getOptions()->getListViewOption() ?>"><?php endif; ?>
+              <span class="title"><?=$this->record($this->driver)->getTitleHtml()?></span>
+            <?php if (!$missing): ?></a><?php endif; ?>
+            <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
+              <div class="title-alt">
+                <?=$this->escapeHtml($altTitle)?>
+              </div>
+            <?php endforeach; ?>
+          </h2>
 
-        <div class="resultItemLine2">
           <?php if ($this->driver->isCollection()): ?>
-            <?=implode('<br>', array_map([$this, 'escapeHtml'], $this->driver->getSummary())); ?>
+            </header>
+            <div class="resultItemLine1">
```

Comment:
We seem to be somewhat redefining the use of the resultItemLine1 and resultItemLine2 classes by moving things around here. Did you investigate whether these are used anywhere in existing styles/code? If not, and if we're willing to break backward compatibility in cases where people have introduced local custom styling against these classes, I wonder if we should revisit the whole naming concept behind them.

Reply:
Well, I didn't find the class names in any styling files or visual differences. Seems like only the lines are numbered by it.

Reply:
I'm pretty sure you're correct that these classes are not used in the core code, but I think some institutions use them to apply local customizations... so I'm concerned that if we change how they are used, it will create weird side effects, and we might be better off eliminating them or replacing them with more descriptive names rather than moving them around. Maybe @crhallberg can weigh in on this issue.

Reply:
It seems that these two classes and `.title-alt` are here for customization, but I am surprised none of the three have any core styles. I'm not too worried about customizations, since they would be obvious and easy to fix, but it's not a non-factor.

Reply:
My thought is mainly that I don't really like the resultItemLine1 / resultItemLine2 class names, as they're not very descriptive... and I'm not sure if they're even true after this refactoring, since resultItemLine1 now comes after the first line of the entry. Maybe we're better off eliminating them or thinking of better names if we're breaking backward compatibility anyway.

Reply:
So I understand that I should remove the `class="resultItemLine1"` here, right @demiankatz ?

Reply:
I just did a search of the code to see where resultItemLine1 is used, and it is present in a number of templates... It is also used in some of our integration tests to identify links. I'm pretty sure that moving or removing it in this template will actually cause tests to fail.

In the long term, I would like to replace this with something more semantically meaningful, and make sure it is applied consistently across different templates. However, that's probably out of scope for this pull request.

Would a better solution be to move the resultItemLine1 class up to the `<header>` element, since that IS now the first line of the result? I believe doing that will at least prevent tests from failing (since the tests are looking for links inside the resultItemLine1 area in order to click on favorites).

Reply:
@demiankatz I did add `class="resultItemLine1"` to header and `class="resultItemLine2"` to the following line